src/PhpPact/FFI/ProxyInterface.php

```diff
@@ -1,8 +1,8 @@
 <?php
 
-namespace PhpPact\Consumer\Service;
+namespace PhpPact\FFI;
 
-interface FFIInterface
+interface ProxyInterface
```

Comment:
What do you think about naming that simply `ClientInterface` -> `Client`? It's in the namespace `FFI` but `proxy` means forwarding something through something before a real target. Here we have an example of the client layer of something.

Instead of that, the Proxy could do some extra logic before passing the request to the client but still can implement `ClientInterface`.

Reply:
Naming is hard for me. Yes, I think `Client` is better than `Proxy`. I renamed as suggestion.